components/Button.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+   <button :class ="[
+      primary ? (disabled ? 'primary-disabled' : 'primary-default') : (disabled ? 'secondary-disabled' : 'secondary-default'), 
+      'baseButton',
+      size === 'small' ? 'small' : '',
+      size === 'medium' ? 'medium' : '',
+      size === 'large' ? 'large' : '',
+      ]">{{ buttonText }}</button>
+</template>
+
+<script>
```

Comment:
Since we moved to Vue 3, we want to start using Composition API and TypeScript. So this should be `<script setup lang="ts">`. This means you'll have to define props with `defineProps` instead. 

See our Vue workshop or https://vuejs.org/guide/components/props.html for examples.